contrib/babelfishpg_common/Makefile

```diff
@@ -79,7 +78,7 @@ include $(PGXS)
 
 ifeq ($(GE91),yes)
 all: sql/$(EXTENSION)--$(EXTVERSION).sql $(UPGRADES)
-all: sql/upgrades/$(EXTENSION)--$(PREEXTVERSION)--$(EXTVERSION).sql sql/upgrades/babelfishpg_upgrades.in
```

Comment:
I don't fully understand why this is a required change. Is there risk of breaking old upgrade test/paths by changing this Makefile?

Reply:
Actually, for upgrade files now we need to add both changes related to spatial datatypes and other datatypes, but if we don't enable the spatial Datatype switch then we don't want these changes in upgrade files so that's why we have created 3 extra files - spatial_types--3.2.0--3.3.0.sql, other_types--3.2.0--3.3.0.sql, babelfishpg_upgrades--3.2.0--3.3.0.in, so that we can automate upgrade file creation through this MakeFile code, and there's no risk of breaking old upgrade test/paths, this is a seperate section which will not affect the old code